policies/ingress/unique_ingress_host.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 Cruise LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//    https://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.package ingress
+
+package ingress
+
+import (
+	"context"
+	"flag"
+
+	"github.com/cruise-automation/k-rail/policies"
+	"github.com/cruise-automation/k-rail/resource"
+	log "github.com/sirupsen/logrus"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func NewPolicyRequireUniqueHost() (PolicyRequireUniqueHost, error) {
+	p := PolicyRequireUniqueHost{}
+
+	var kubeconfig = flag.String("kubeConfig", "", "absolute path to the kubeconfig file: `<home>/.kube/config`")
+	flag.Parse()
```

Comment:
Benchmark is currently failing because the default flag package cannot have the flag declared multiple times, which happens as the benchmarks are run. The solution here is to move the flag definition to the entrypoint or provide the configuration through the existing config file that is used. 
Our deployment uses the in cluster config though, so maybe we should just omit this flag altogether and pass an empty string to BuildConfigFromFlags below.